src/Data/RFC1524.hs

```diff
@@ -27,7 +27,7 @@ import Data.RFC1524.ViewCommand
 type MailcapFile = [MailcapLine]
 
 data MailcapLine
-  = Comment B.ByteString
+  = Comment
```

Comment:
Can't we remove the `Comment` constructor entirely?

Reply:
You know... I tried to think of a good excuse, but can't find any ... Will remove